### PR TITLE
Pipeline work flow integration for node

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN pushd /coreapi && \
     # needed for DB migrations
     find coreapi/ -mindepth 1 -maxdepth 1 \( ! -name 'alembic*' -a ! -name hack \) -exec rm -rf {} +
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+RUN pip3 install git+https://github.com/yzainee/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
```
Comment (miteshvp) [inline Dockerfile]: this will pull in the changes from your repo. It should pick from upstream repo. Revert the changes in this line

Comment (samuzzal-choudhury) [inline Dockerfile]: @yzainee install has to point to the `fabric8-analytics` repo.

Comment (samuzzal-choudhury) [reply Dockerfile]: @yzainee please verify the worker commit hash in the Dockerfile and Dockerfile.rhel as well

Comment (yzainee) [reply Dockerfile]: Typo.. rectified

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -832,33 +812,55 @@ def post():
...
             d = DependencyFinder()
-            if origin == "vscode" and ecosystem in ("npm", "pypi"):
+            deps = {}
+            worker_flow_enabled = "false"
+
+            if resolved_files_exist(manifests) == "true" \
+                    and ecosystem in ("npm", "pypi"):
                 deps = d.scan_and_find_dependencies(ecosystem, manifests)
+            elif scan_repo_url and ecosystem == "npm":
+
```
Comment (miteshvp) [inline bayesian/api_v1.py]: All these conditions should mention what it is doing
e.g. this elif is we get a git url . Add TODO that says, add support for other ecosystems as well. Currently it only supports npm

File: bayesian/api_v1.py [hunk 3]
```diff
@@ -832,33 +812,55 @@ def post():
...
+                        'is_scan_enabled': is_scan_enabled,
+                        'request_id': request_id,
+                        'is_modified_flag': is_modified_flag,
+                        'auth_key': request.headers.get('Authorization'),
+                        'check_license': check_license,
+                        'gh_token': github_token
+                        }
+                server_run_flow('gitOperationsFlow', args)
+                worker_flow_enabled = "true"
             else:
```
Comment (miteshvp) [inline bayesian/api_v1.py]: add comment that says call mercator and identify dependencies
